fitbenchmarking/fitbenchmarking
# Remove duplication of Bennett5 in CUTEST

#### Description of Work

Fixes #317 -- removed the two offending problems, which were just duplicates
of the NIST problem Bennett5 with a name change

#### Testing Instructions

1. Check nothing else has changed in the code base(?)

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
